up-l1/zenoh.adoc

```diff
@@ -86,82 +96,107 @@ Different message types **MUST** use different Zenoh API.
+  ue_version_major: 0x03,           // version 3
+  resource_id: 0x800A               // resource 0x800A
+}
+----
+
+The zenoh_key should be `up/192.168.1.100/empty/10002/3/800A`
+
+* Notification
 
-[source]
+[example]
 ----
-UUri {
-  authority: empty,
-  entity: UEntity {
-    name: "body.access",
-    version_major: 1,
-    id: 1234,
-  }
-  resource: UResource {
-    name: "door",
-    instance: "front_left",
-    message: "Door",
-    id: 5678,
-  }
+Source UUri {
+  authority_name: "192.168.1.99",
+  .. // Not important
+}
+Destination UUri {
+  authority_name: "192.168.1.100",
+  ue_id: 0x0001_0002                // instance 1, service 2
+  ue_version_major: 0x03,           // version 3
+  resource_id: 0x800A               // resource 0x800A
+}
+----
+
+The zenoh_key should be `up/192.168.1.99/192.168.1.100/10002/3/800A`
+
+* Request
+
+[example]
+----
+Source UUri {
+  authority_name: "192.168.1.99",
+  .. // Not important
+}
+Destination UUri {
+  authority_name: "192.168.1.100",
+  ue_id: 0x0001_0002                // instance 1, service 2
+  ue_version_major: 0x03,           // version 3
+  resource_id: 0x000B               // resource B
 }
 ----
 
-It's a local UUri, so it starts with `upl`.
-Also, we leave the second part blank since it doesn't have UAuthority.
-Only ID in UEntity and UResource matters while serializing into link:../basics/uri.adoc#421-local-micro-uri[micro-form Uri].
-Then transform it into lowercase hexadecimal representation.
-The Zenoh key is `upl/0100162e04d20100`.
+The zenoh_key should be `up/192.168.1.99/192.168.1.100/10002/3/B`
 
-* The remote UUri with UAuthority
+* Special UUri
 
-[source]
+[example]
 ----
-UUri {
-  authority: UAuthority {
-    name: UAuthName,
-    number: Id({01, 02, 03, 10, 11, 12})
-  },
-  entity: empty,
-  resource: empty,
+Source UUri {
+  authority_name: "*",
+  .. // Not important
+}
+Destination UUri {
+  authority_name: "192.168.1.100",
+  ue_id: 0x0000_FFFF                // any instance, any service
+  ue_version_major: 0xFF,           // any version
+  resource_id: 0xFFFF               // any resource
 }
 ----
 
-It's a remote UUri, so it starts with `upr`.
-The UAuthority is serialized into mirco form.
-Given how a UAuthority containing an ID is serialized into micro form as outlined in link:../basics/uri.adoc#422-remote-micro-uris[the spec],
-it is length + serialized UUri in lowercase hexadecimal representation, which is `06` + `0102030a0b0c`.
-The UEntity and UResource are all blank, so it is the special form UUri.
-It means that the third part is `+++**+++`.
-Therefore, the Zenoh key is `upr/060102030a0b0c/+++**+++`.
+This means we want to register the messages to a certain destination.
+The zenoh_key should be `up/+++*+++/192.168.1.100/+++*+++/+++*+++/+++*+++`
 
 === UPayload
 
-Although UPayload can be encoded into protobuf, it saves a lot of time without encoding.
-UPayload consists of three parts: length, format, data.
+UPayload consists of two parts: data, format.
```

Comment:
Please see recent changes, there is no more UPayload. All we have is payload in UMessage and the UPayloadFormat is in UAttributes

Reply:
OK, it's merged now. Then I'll update it.